app/assets/javascripts/components/GlobalNav.js

```diff
@@ -0,0 +1,160 @@
+define(['jquery', 'DoughBaseComponent', 'mediaQueries', 'utilities'], function($, DoughBaseComponent, mediaQueries, utilities) {
+  'use strict';
+
+  var GlobalNav;
+
+  GlobalNav = function($el, config) {
+    GlobalNav.baseConstructor.call(this, $el, config);
+
+    this.$globalNav = this.$el;
+    this.$mobileNavButton = $(document).find('[data-dough-mobile-nav-button]');
+    this.$globalNavClumps = this.$el.find('[data-dough-nav-clumps]');
+    this.$globalNavClump = this.$el.find('[data-dough-nav-clump]');
+    this.$globalNavClumpHeading = this.$el.find('[data-dough-nav-clump-heading]');
+    this.$globalSubNavHeading = this.$el.find('[data-dough-subnav-heading]');
+    this.$globalSubNav = this.$el.find('[data-dough-subnav]');
+    this.delay = 250; // ms
+    this.mobileAnimationDuration = this.$globalNav.css('transition-duration');
+  };
+
+  /**
+  * Inherit from base module, for shared methods and interface
+  */
+  DoughBaseComponent.extend(GlobalNav);
+  GlobalNav.componentName = 'GlobalNav';
+
+  /**
+  * Initialize the component
+  */
+  GlobalNav.prototype.init = function(initialised) {
+    this._initialisedSuccess(initialised);
+    this._setUpMobileInteraction();
+    this._setUpDesktopInteraction();
+    this._setUpMobileAnimation();
+
+    $(window).on('resize', utilities.debounce($.proxy(this._resize, this), 100));
+
+    return this;
+  };
+
+  // Ensure mobile nav not activated on resize
+  GlobalNav.prototype._resize = function() {
+    this._setUpMobileAnimation();
+  }
+
+  GlobalNav.prototype._setUpMobileAnimation = function() {
+    if (mediaQueries.atSmallViewport()) {
+      this.$globalNav.css('transition-duration', this.mobileAnimationDuration);
+    } else {
+      this.$globalNav.css('transition-duration', '0s');
```

Comment:
I think we should add / remove a `.no-transition` class instead of manipulating the CSS directly, but as discussed, this might not work as this is getting quite fiddly